test-js/unit/pentaho/type/simple.serialization.Spec.js

```diff
@@ -158,10 +224,13 @@ define([
 
       describe("values", function() {
         describe("primitive format (omitFormatted: true)", function() {
-          it("should return the primitive value", function() {
+          it("should return the primitive value when it is not a plain object or wrap it otherwise", function() {
             var spec = value.toSpec({omitFormatted: true, includeType: false});
-
-            expect(spec).toBe(simpleSpec.value);
+            if(simpleSpec.value instanceof Object && simpleSpec.value.constructor === Object) {
```

Comment:
Multiple paths in a unit test should be avoided.
I understand this is running in the `simplerTypes.forEach` loop, but who to make clear to whoever is reading the code which of the above should or shouldn't be wrapped?